Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -34,7 +35,14 @@ private void OnAfterInteract(EntityUid uid, StickyComponent component, AfterInte
         if (args.Handled || !args.CanReach || args.Target == null)
             return;
 
-        // try stick object to a clicked target entity
+        //makes sure that neither the target nor the host have entities stuck to them or is stuck to entities
```

Comment:
By design, you should be able to stick any amount of entities to one entity. Not sure why do you need this check.

Reply:
I plan to add an item glue that adds the sticky component onto items, I don't want infinite storage exploit so I implemented this check. Of course I can make the component carry an int that accounts for the maxium amount of items that were able to stick to an object

Reply:
Yeah, maybe something sane like only 10 sticky items per entity. This should be probably const in `StickySystem`